frontend/test/resolution/testCatch.cpp

```diff
@@ -445,6 +445,40 @@ static void test12b(Parser* parser) {
 }
 
 
+// after compilerError, try-catch analysis should go and try access unresolved
+// AST.
+static void test12c(Parser* parser) {
+  auto ctx = buildStdContext();
+  auto path = UniqueString::get(ctx, "test12.chpl");
+
+  std::string program = R""""(
+                        module M {
+                          proc test() {
+                            compilerError("oopsie!");
+
+                            // this is a regression test; in the past,
+                            // trying to branch-sensitively traverse this
+                            // loop (which wasn't resolved, since we
```

Comment:
Can you double check this comment, which mentions a loop?

Reply:
good catch, it's a conditional, not a loop.